libs/libcommon/src/libcommon/config.py

```diff
@@ -143,7 +143,9 @@ def from_env() -> "WorkerLoopConfig":
 class ProcessingGraphConfig:
     specification: ProcessingGraphSpecification = field(
         default_factory=lambda: {
-            "/splits": {"input_type": "dataset", "required_by_dataset_viewer": True},
+            "/config-names": {"input_type": "dataset"},
+            "/split-names": {"input_type": "config", "requires": "/config-names"},
+            "/splits": {"input_type": "dataset", "required_by_dataset_viewer": True},  # to be deprecated
```

Comment:
This is where you define the dependencies between the processing steps ?

Reply:
Yes! It's a bit hidden for now, and it's hard-coded even if it seems to be a config option. I always thought of this as a temporal workaround, but I'm unsure how we want this to evolve.